odin-control.md

```diff
@@ -137,63 +137,98 @@ logic on UI side, do it in the adapter
 
 ## Getting started demo
 
-* Create a python 2.7 virtual environment (various methods):
+* For this demo you will need to have Python 3.X installed on your machine.
```

Comment:
Tornado 6 (latest version) requires python >= 3.5 - might want to mention that.

Reply:
Good point, thanks.